qutip/qobjevo.py

```diff
@@ -217,21 +230,23 @@ def __getitem__(self, i):
 
 
 class QobjEvo:
-    """A class for representing time-dependent quantum objects,
-    such as quantum operators and states.
-
-    The QobjEvo class is a representation of time-dependent Qutip quantum
-    objects (Qobj). This class implements math operations :
-        +,- : QobjEvo, Qobj
-        * : Qobj, C-number
-        / : C-number
-    and some common linear operator/state operations. The QobjEvo
```

Comment:
This paragraph seems to have been entirely dropped. It wasn't that clear to me what the paragraph was saying, so perhaps dropping it was not a bad thing, just checking that doing so was intentional.

Reply:
It's still there in my version, isn't it? It's just reformatted into a list.  The method documentation is all "removed" because it reality it's generated into the methods individually by `numpydoc`.

Reply:
I can see the list, but the text:

> and some common linear operator/state operations. The QobjEvo are constructed from a nested list of Qobj with their time-dependent coefficients. The time-dependent coefficients are either a funciton, a string or a numpy array.

seems to be gone (or at least I couldn't find it with a search).

Reply:
Ah, thanks for clarifying - the way this comment's appear, I didn't see the extra bit you were talking about.  It _does_ seem like I've lost a sentence - I'll add it back in.

Reply:
Added bit looks good now.